### DISCUSSION TITLE
Future Plans

### DISCUSSION THREAD

**Discussion by mstamy2** [py-pdf/pypdf#686] [General]
Hello all. I'm working to find a [solution to keep active maintenance going on this project.](https://github.com/mstamy2/PyPDF2/wiki/State-of-PyPDF2-and-Future-Plans)

Feedback and discussion is appreciated!

**Comment by alexis-via** [comment]
I think you should at least grant release rights on pypi for PyPDF2 to some of your biggest and most serious contributors on PyPDF2, so that someone can rapidly push a release to pypi using the current master branch. There are important things in master which are not present in the last release on Pypi, and it's a real problem when you have to explain to non-expert users that they should do a git checkout then install the lib manually, etc...

**Comment by kurtmckee** [comment]
@mstamy2 I don't have a recommendation for active maintenance, but I wanted to let you know that I'm working on a significant merge request to add a lot of unit tests. Thanks for your work on PyPDF2!

**Comment by dreua** [comment]
Maybe have a look at https://www.codeshelter.co/ for help. 

I believe it would be best if all development would go into PyPdf2 (and not in any forks), so everyone using it, e.g. as a package in many linux distributions, benefits from fixes and improvements.

**Comment by alexis-via** [comment]
@mstamy2 Have you taken a decision about this ?

**Comment by Rob1080** [comment]
@mstamy2 any updates regarding your plans?
@claird, I looked at pypdf4 but it also seems to have stalled.
